setup.cfg

```diff
@@ -72,13 +72,13 @@ install_requires =
     protobuf>=4.21.2,<4.22.0
     psutil>=5.6.0
     pyzmq==22.*
-    setuptools>=49
+    setuptools>=49; python_version > "3.7"
+    setuptools>=49, <67; python_version == "3.7"
```

Comment:
Ah actually I have just looked at each fast tests' logs and found that for Python 3.8-10, it's using a cached version of setuptools that is lower than 67 anyway, so maybe the problem will still be present for those versions too

Reply:
Yep, problem present on all versions of Python unfortunately: https://github.com/MetRonnie/cylc-flow/actions/runs/4043433284/jobs/6952341119